serverless-plugin-db-backups.js

```diff
@@ -115,28 +108,33 @@ class DynamodbAutoBackup {
     }
 
     if (!has(this.dynamodbAutoBackups, 'slackWebhook')) {
-      DynamodbAutoBackup.consoleLog('  sls-plugin-backup: -----------------------------------------------------------');
-      DynamodbAutoBackup.consoleLog('         Warning: slackWebhook is not provide, you will not be notified of errors !');
+      console.log(chalk.yellow.bold('@unly/serverless-plugin-db-backups: -----------------------------------------------------------'));
+      console.log('         Warning: slackWebhook is not provide, you will not be notified of errors !');
+      console.log();
     }
 
+    this.functionBackup = {
+      name: this.dynamodbAutoBackups.name || `${this.serverless.service.service}-${this.serverless.service.provider.stage}-dynamodbAutoBackups`,
+      handler: this.dynamodbAutoBackups.source,
+      events: [],
+      environment: {},
+    };
+
     return BbPromise.resolve();
   }
 
   constructFunctionObject() {
+    const events = [];
 
     if (isString(this.dynamodbAutoBackups.backupRate)) {
       const cron = {
         schedule: this.dynamodbAutoBackups.backupRate,
       };
-      this.functionBackup.events.push(cron);
-    }
-
-    if (has(this.dynamodbAutoBackups, 'name') || isString(this.dynamodbAutoBackups.name)) {
-      assign(this.functionBackup, this.dynamodbAutoBackups.name);
+      events.push(cron);
     }
 
-    if (includes(this.custom, 'development')) {
-      this.functionBackup.events.push(
+    if (!includes(this.serverless.service.provider.stage, 'production')) {
```

Comment:
What does this do? Add a HTTP event to perform backup in a non-production environment?
Should be documented here (and in readme too, under the `Contributors` section)

Is that how you perform your local testing?